subsys/bluetooth/controller/Kconfig

```
@@ -163,19 +163,23 @@ config BT_CTLR_SDC_TX_PACKET_COUNT
 	range 1 20
 	help
 	  The number Link Layer ACL TX packets reserved per connection.
-	  With the default count, the application is able to refill the buffers during a connection event.
-	  That is, non-default values should only be used if reduced throughput is accepted, or when the CPU
-	  utilization is so high that the application is not able to provide data fast enough during connection events.
+	  With the default count, the application is expected to be able to refill
+	  the buffers during a connection event. That is, non-default (>3) should only
```

```suggestion
	  the buffers during a connection event. That is, non-default values (>3) should only
```
we should either put it back here or remove from line 179